general/README.md

```diff
@@ -218,5 +218,58 @@ _handleEvent = function(e) {
     _doSomethingWithEvent(checked);   
 }
 ```
+### Single returns
 
+If possible, avoid using more than one return statement inside a given function. Instead, declare a local variable to reassign the value you want to return. It makes it easier for maintainability and avoids mixing decision with execution.
```

Comment:
This rule kind of conflicts with the "multiple `if-else`" rule earlier. In that example we suggest using an object literal look-up in order to separate decision w/ execution. This one is a bit trickier given that the conditions are all very different.